react/admin/settings/RMASettings.tsx

```diff
@@ -71,9 +107,53 @@ export const RMASettings = () => {
       return
     }
 
+    if (!isAnyPaymentMethod && enablePaymentMethodSelection) {
+      setHasPaymentMethodError(true)
+
+      return
+    }
+
+    setHasPaymentMethodError(false)
+
     handleSaveAppSettings()
   }
 
+  const handleOptionSelection = (options: { string: CheckboxProps }) => {
+    const paymentOptions = Object.keys(options)
+
+    const updatedPaymentOptions = paymentOptions.reduce<PaymentType>(
+      (acc, paymentOption) => {
+        return {
+          ...acc,
+          [paymentOption]: options[paymentOption].checked,
+        }
+      },
+      {}
+    )
+
+    const paymentOptionsPayload = {
+      ...appSettings.paymentOptions,
+      allowedPaymentTypes: updatedPaymentOptions,
+    }
+
+    dispatch({
+      type: 'updatePaymentOptions',
+      payload: {
+        ...appSettings.paymentOptions,
+        allowedPaymentTypes: updatedPaymentOptions,
+      },
+    })
+
+    if (!validateOptions(paymentOptionsPayload)) {
+      setHasPaymentMethodError(true)
+      window.scrollTo(0, 20)
```

Comment:
This is causing a weird behavior. Every time I unselect all the page jumps up. Maybe you want to add this scroll when the user submits, not when they select the option.